docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Rerunning Introspection"
+date: 2020-06-04T08:14:51-05:00
+draft: true
+weight: 5
+description: "This document describes rerunning the introspector job in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+
+This document describes _how and when_ to rerun the introspector job in the Oracle WebLogic Server in Kubernetes environment.
+
+#### Overview
+
+In order to manage a domain, the operator needs to know its definition in WebLogic Server. It does this by running an _introspector job_
+on the domain definition when the domain is first deployed in the operator. If the configuration is modified, the operator must rerun
+the _introspector job_. The exact behavior will depend on the model chosen, and the detected change. Often the running
+domain can be updated without restarting any servers. Currently, this is not supported for Model in Image.
+
+#### Common re-introspection scenarios
+
+This document describes what actions allow avoiding the restart of servers in a number of common scenarios:
```

Comment:
common scenarios -> common re-introspection scenarios (the reason I'm suggesting repeating "re-introspection" here is because the use cases focus on which scenarios may or may not require server restarts, but they all invoke re-introspection, right? I was losing that thought and then I got to the "how to force the introspector job" section and came back to the focus of this doc, which is actions that cause rerunning the introspector job.